modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -742,12 +799,15 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
                                         memset(rowbuf, 0, vsz*sizeof(rowbuf[0]));
                                         for( k = 0; k < ncn; k++ )
                                         {
-                                            for( i = i0; i < i1; i++ )
+                                            for (int d = d0; d < d1; d++)
```

Comment:
What about moving definition of `d` to other spatial variables `int i, j, k;`? Maybe it can solve tails processing to enable `avx2`?

Reply:
k - `ncn` counter

Reply:
Yeah, I see. I mean `i` and `j` are spatial counters (height and width). So due depth is also spatial dimension, they can be equally used?